code/modules/nano/nanoui.dm

```diff
@@ -511,7 +511,7 @@ nanoui is used to open and update nano browser uis
 		var/newz = input("Choose Z-Level to view.","Z-Levels",1) as null|anything in list(1,3,4,5,6)
 		if(!newz || isnull(newz))
 			return 0
-		if(newz < 1 || newz > 6 || newz == 2)
+		if(newz < 1 || newz > 6 || newz == map.zCentcomm)
```

Comment:
Don't see what needs to be changed? The intent is that it shows maps for anything that isn't centcomm and isn't a vault level, I think? Which is what it does